docs/db/db-authorization.md

```diff
@@ -0,0 +1,229 @@
+# DB authorization
+
+Every request to Hasura executes against a set of session variables. Normally there are some main variables for the authorization context:
+
+- `X-Hasura-User-Id`: this variable usually denotes the user executing the request.
+
+- `X-Hasura-Role`: This variable denotes the role with which the user is executing the current request. Hasura has a built-in notion of a role, and will explicitly look for this variable to infer the role.
+
+- `X-Hasura-campus`: this variable contains the campus that the user currently is on.
```

Comment:
Isn't it `x-hasura-campuses`? I think it list all the campuses the user has access to (for admin campus accesses isn't it?)